incubator/tensorflow-inception/templates/inception-job.yaml

```diff
@@ -0,0 +1,31 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: {{ template "fullname" . }}-seed-inception
+  labels:
+    app: {{ template "fullname" . }}
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+    release: "{{ .Release.Name }}"
+    heritage: "{{ .Release.Service }}"
+spec:
+  template:
+    metadata:
+      name: seed-inception
+    spec:
+      containers:
+      - name: seed
+        image: {{ .Values.client.image }}
+        command: ["/bin/sh"]
+        args: ["-c", "curl -o /seed/inception-v3-2016-03-01.tar.gz http://download.tensorflow.org/models/image/imagenet/inception-v3-2016-03-01.tar.gz && \
```

Comment:
I think this will be easier to read if you do it like https://github.com/kubernetes/charts/blob/master/incubator/etcd/templates/etcd-statefulset.yaml#L92